src/basic/meson.build

```diff
@@ -91,6 +91,8 @@ basic_sources = files('''
         format-util.h
         fs-util.c
         fs-util.h
+        gcrypt-util.c
+        gcrypt-util.h
```

Comment:
i don't follow here... Aren't we pulling in a libgcrypt dep into everything that uses src/basic/ now?